frontend/src/components/dashboard/minesHomePage/Dashboard.js

```diff
@@ -156,9 +161,15 @@ export class Dashboard extends Component {
         ...remainingParams,
       },
     });
-    this.props.fetchMineRecords(params).then(() => {
-      this.setState({ mineList: true });
-    });
+    if (remainingParams.map) {
+      this.props.fetchMineRecordsForMap().then(() => {
+        this.setState({ mineList: true });
+      });
+    } else {
+      this.props.fetchMineRecords(params).then(() => {
+        this.setState({ mineList: true });
```

Comment:
Good catch! 
From what I can see, the `map=true` param is still useful, just in a different way than it was being used before. (ie frontend check vs backend) I can see how it will be confusing going forward. 

Better approaches:
Remove that param from the url, only use the `this.fetchDataFromUrl` to update the mineList using `this.props.fetchMineRecords`

`this.props.fetchMineRecordsForMap` could be run in the background on `componentDidMount()`, and stored in a different const in the mineReducer, - The benefit here, is that the user will not have to wait for all the mine information to be fetched when they navigate to the map view.